docs/projects/ad7606x_fmcz/index.rst

```diff
@@ -0,0 +1,450 @@
+=============== === ========== ===========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <main:/>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository, and then build the project as follows:.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/ad7606x_fmc/zed
+   user@analog:~/hdl/projects/ad7606x_fmc/zed$ make DEV_CONFIG=2 INTF=0
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make DEV_CONFIG=2 INTF=0``
+
+then the folder name will be:
+
+``DEVCONFIG2_INTF0``
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Connections and hardware changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   **The following hardware changes are required:**
+
+   Depending on the required interface mode, some hardware modifications need to
+   be done.
+
+   -   **JP5** - A - Serial interface
+   -   **JP5** - B - Parallel interface
+   -   **JP7, JP12, JP13, JP14** - B - Differential operation
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :dokuwiki:`AD7606X HDL Reference Design <resources/eval/user-guides/ad7606x-fmc/hdl>`
+-  :dokuwiki:`AXI_AD7616 <resources/fpga/docs/axi_ad7606x>`
+-  :dokuwiki:`AD7606-No-OS Driver <resources/tools-software/uc-drivers/ad7606>`
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheet: :adi:`AD7606`
+
+-  `UG-1870, Evaluation Board User Guide <https://www.analog.com/media/en/technical-documentation/user-guides/eval-ad7606c-fmcz-ug-1870.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`ad7616x_fmc HDL project <main:projects/ad7606x_fmc>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AD_EDGE_DETECT
+     - :git-hdl:`library/common/ad_edge_detect.v <main:library/common/ad_edge_detect.v>`
+     -
+   * - AXI_AD7606
+     - :git-hdl:`library/axi_ad7606x <main:library/axi_ad7606x>` *
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_ad7606x>`
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_dmac <main:library/axi_clkgen>` **
```

Comment:
axi_clkgen isn't used for both Serial and Parallel? If so remove the "**" at the end

Reply:
Done.